keyboards/bajjak/readme.md

```diff
@@ -0,0 +1,2 @@
+# BAJJAK KEYBOARD
+* This keyboard is a 6x7 array keyboard.
```

Comment:
This needs to be filled out, using the default template. 

https://github.com/qmk/qmk_firmware/blob/e49fa43a7f4025f60ff61ff2fa5015a8fec7ddac/data/templates/keyboard/readme.md